### ISSUE TITLE
[Redmi Note 7] Audio jack not working with high impedance headphones.

### ISSUE THREAD

**Issue by datspike** [phhusson/treble_experimentations#460]
Redmi Note 7 global, stock /boot and /vendor partitions from last china dev rom, Magisk 19.0 installed as device does not boot without it for some reason.
Audio jack is not working with high impedance headphones (HyperX Cloud Core), but is working with basic 16 Ohm headphones. 
I'm able to switch to headphones using [this app](https://play.google.com/store/apps/details?id=com.nordskog.LesserAudioSwitch)
No audio even if booting the system with high impedance headphones plugged in.
[getprop on gsi](https://gist.github.com/datspike/223e7477614a39e68ea4c8561537b51f)
Gists with high impedance headphones:
[getevents plug\unplug](https://gist.github.com/datspike/a28c7e5d72ad0008005f441a5a83ddc7)
[logcat](https://gist.github.com/datspike/f3c09bf1cf4a6485332174773b7eee0d)
Gists with low impedance headphones:
[getevents plug\unplug](https://gist.github.com/datspike/644cbedae9d1a5ba06bdad2b221c2981)
[logcat](https://gist.github.com/datspike/fc042417a0f64b53444a04bce203a33a)

**Comment by phhusson** [comment]
uh, does it work on stock ROM?

Le jeu. 4 avr. 2019 à 21:24, Elesin Alexey <notifications@github.com> a
écrit :

**Comment by datspike** [comment]
Yes it does. I will provide a getprop log from stock rom in ~12 hours

пт, 5 апр. 2019 г., 1:27 Pierre-Hugues HUSSON <notifications@github.com>:

**Comment by datspike** [comment]
Getprop on miui: [getprop](https://gist.github.com/datspike/a86ac05289334256ba1008ed22442e51)
